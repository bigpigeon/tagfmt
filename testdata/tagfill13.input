//tagfmt -f "json=or(:tag,'')"

package main

type OrderDetail struct {
    ID       string   `json:"Id"`
    UserName string   `json:"User"`
    OrderID  string   ``
    Callback string   ``
    Address  []string ``
}
