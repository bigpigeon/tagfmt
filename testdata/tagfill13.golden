//tagfmt -f "json=or(:tag,'')"

package main

type OrderDetail struct {
	ID       string   `json:"Id"`
	UserName string   `json:"User"`
	OrderID  string   `json:""`
	Callback string   `json:""`
	Address  []string `json:""`
}
